loader/src/main/java/net/neoforged/fml/loading/ModDirTransformerDiscoverer.java

```diff
@@ -93,9 +90,6 @@ private static boolean shouldLoadInServiceLayer(Path path) {
         if (!path.toString().endsWith(".jar")) return false;
         if (LamdbaExceptionUtils.uncheck(() -> Files.size(path)) == 0) return false;
 
-        JarMetadata metadata = JarMetadata.from(new JarContentsBuilder().paths(path).build());
-        return metadata.providers().stream()
-            .map(SecureJar.Provider::serviceName)
-            .anyMatch(TransformerDiscovererConstants.SERVICES::contains);
+        return TransformerDiscovererConstants.shouldLoadInServiceLayer(path);
```

Comment:
Now that we're delegating to this method, is `shouldLoadInServiceLayer` a good name for this method? Would something like `isValidServiceLayerJar` be a better name?

Reply:
I think it's fine like this. Find a synonymous name doesn't make the method clearer.